lms/views/lti_launches.py

```diff
@@ -56,8 +57,13 @@ def is_db_configured(request, params):
     stored in our database. This occurs when an lms does not support content
     item selection
     """
+    try:
+        resource_link_id = params['resource_link_id']
+    except KeyError:
+        raise MissingLTILaunchParamError('Resource link id is required for lti launch.')
```

Comment:
A couple of tweaks here:

1) Can you run the string through i18n? (i.e. the `_` function)?
2) Can we use the precise param name in this (technical) error message? e.g. `"LTI data parameter resource_link_id is required for launch"`